src/components/HomeSection.test.jsx

```diff
@@ -41,4 +42,30 @@ describe('HomeSection', () => {
     const items = screen.queryByRole('button');
     expect(items).toBe(null);
   });
+    
+    test('render given buttons', async () => {
+        render(
+            <HomeSection
+              title="Topology"
+              description="Math course"
+              buttons={[
+                  <Button
+                    to="/about"
+                    variant="outlined"
+                    color="primary">
+                    Hello
+                  </Button>,
+                  <Button
+                    to="/about"
+                    variant="outlined"
+                    color="primary">
+                    Hello
+                  </Button>,
```

Comment:
You can simplify this by removing the optional properties:
```suggestion
                  <Button>Hello</Button>,
                  <Button>Hello</Button>,
```

You're not testing against those property values anyways.